org.simondalvai.ball2box.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.simondalvai.ball2box
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.godotengine.godot.BaseApp
+base-version: '3.5'
+command: godot-runner
+finish-args:
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+modules:
+  - name: Ball2Box
+    buildsystem: simple
+    build-commands:
+      # The game itself
+      - install Ball2Box.pck /app/bin/godot-runner.pck
```

Comment:
You aren't installing the executable from the archive?

Reply:
```
% flatpak run org.godotengine.godot.BaseApp//3.5 
Error: Couldn't load project data at path ".". Is the .pck file missing?
If you've renamed the executable, the associated .pck file should also be renamed to match the executable's name (without the extension).
```

So this indicate that 
1. it will work
2. it will also work on aarch64 which shouldn't be disabled.

Reply:
I just copied the build-commands from Inifinte Shooter:
https://github.com/flathub/io.github.pastthepixels.InfiniteShooter/blob/master/io.github.pastthepixels.InfiniteShooter.yml

Is the executable needed?

Reply:
As I said it doesn't seem to be since it's using the base app executable.